### ISSUE TITLE
[Feature Request] Any way to get the mesh?

### ISSUE THREAD

**Issue by MoonMoon82** [deroberon/StableZero123-comfyui#3]
It would be awesome to get a 3d model to export in any file format like obj, stl or something like that.
Does the output need to be uploaded to a photogrammetry program?

**Comment by deroberon** [comment]
Yes, it would be amazing but right now the model implemented just give us images. I tought about the photogrammetry also, but I don't know if the angles we have (and the precision in the generation) will allow a good 3d reconstruction.

**Comment by Dobidop** [comment]
Not sure how closely this relates to https://github.com/threestudio-project/threestudio but maybe some of the needed code can be lifted from that?

**Comment by deroberon** [comment]
Humm... It seems interesting. I have to stop and try to understand the code to see what can be done.
